### ISSUE TITLE
scalability plots with different SIMD and different numbers of cores used

### ISSUE THREAD

**Issue by valassi** [madgraph5/madgraph4gpu#519]
Hi all, it would be nice to have some scalability plots with different SIMD and different numbers of cores used

I produced some for ggttgg in cudacpp in MR #518 

![image](https://user-images.githubusercontent.com/3473550/187084056-ae50f51e-27e3-4cad-b4a9-622e447c5122.png)

@jtchilders @nscottnichols @whhopkins do you think it would be possible to create something similar for kokkos and sycl? Here for cudacpp I am doing several copies in parallel as I have no MT (I disabled OMP). For kokkos and sycl the question that we discussed in some meetings is whether/how you can switch on/off simd levels (and if they do anything at all) and also how to modify the numbers of CPU cores that are used.

Thanks!
Andrea

**Comment by valassi** [comment]
The plots above were obtained manually, by building the cudacpp code and then executing it with ad hoc scripts.

In MR #527 I have now added a set of scripts and plots based on containers from the BMK project. This makes it MUCH easier to extract data. 

Note: the fact that avx2 above seens to exploit HT and even overcommit is probably an artefact. The tests are too short, then they do not overlap with one another long enough. I definitelty saw this in other places (eg ggttg are configured there to run too short, and the effect is much more visible). I will evevntually rerun with much longer durations...

**Comment by valassi** [comment]
For cudacpp, this is what I have done here
https://indico.cern.ch/event/1170924/contributions/4954511/
![image](https://user-images.githubusercontent.com/3473550/195983921-cf1d109a-d33a-40a7-a838-67c09f2dfd58.png)
